s3/pkg/service/bucket.go

```diff
@@ -84,18 +84,21 @@ func (s *s3Service) CreateBucket(ctx context.Context, in *pb.Bucket, out *pb.Bas
 		return err
 	}
 
-	sd, err := driver.CreateStorageDriver(backend.Type, backend)
-	if err != nil {
-		log.Errorln("failed to create storage. err:", err)
-		return err
-	}
+	if backend.Type == "aws-s3" || backend.Type == "azure-blob" || backend.Type == "gcp-s3" ||
```

Comment:
Can we improve on this condition, something like backend is cloud Type then condition is true

Reply:
done. Thanks